shared/cfg/machines.cfg

```diff
@@ -29,6 +29,13 @@ variants:
         only ppc64, ppc64le
         machine_type = pseries
         # Currently only supports standard VGA
+        # Make sure below param is set 'yes' to run a
+        # power8(compat mode) on power9 host
+        # power9_compatfixup = "no"
```

Comment:
```
# power8 guest on power9 (compat mode) host
# power9_compat = "no"
```

Reply:
@balamuruhans hope this is going to be a workaround to enable compat mode, according to mentioned linux patch, so used that naming convention.